src/pynumerals/commands/create_language_sheet.py

```diff
@@ -0,0 +1,141 @@
+"""
+Creates Excel file in raw/xlsx having two sheets 'data' and 'metadata'
+based on data of the passed language ID.
+If passed language ID is 'template' empty sheets of all parameter will be generated.
+"""
+import pathlib
+import openpyxl
+import collections
+import pycldf
+import pylexibank
+
+from cldfbench.cli_util import with_dataset, add_dataset_spec
+from ..numerals_utils import XLSX_LABELS
+
+
+def register(parser):
+    add_dataset_spec(parser)
+    parser.add_argument(
+        '--lang-id',
+        type=str,
+        help='CDLF Language_ID',
+        default=None,
+    )
+
+
+def run(args):
+    with_dataset(args, create_language_sheet)
+
+
+def create_language_sheet(dataset, args):
+    assert args.lang_id
+    is_template = False
+    ds = dataset.cldf_reader()
+
+    desired_lang_id = str(args.lang_id)
```

Comment:
Why do we need to cast to `str` here? That should have been done by `argparse` already.